pkg/api/gitops/v1alpha1/constants.go

```diff
@@ -36,3 +36,5 @@ const ApplicationFinalizerName = "application." + GroupName
 
 // ArgoCDResourcesFinalizer is the name of Argo CD resource finalizer
 const ArgoCDResourcesFinalizer = "resources-finalizer.argocd.argoproj.io"
+
+const ArtifactRepoLabelKey = GroupName + "/isArtifactRepo"
```

Comment:
perfer to use `is-artifact-repo`

Reply:
I actually have a question about the label style.

1. `devops.kubesphere.io/is-artifact-repo: true` the value `true` here is a string type.
2. `devops.kubesphere.io/repository-type: Artifact | Non-Artifact`

Are the both option ok? or Is there a convention about the label style?

Reply:
I think we should use only one label key to mark the repo. In this way, there are only three situations: existence and the value is true,  existence and the value is false or non-existence.

In fact, there is no convenient way to convert. Here is a pseudocode from other open source projects

```
func GetStringLabel(name string) string {
	return labels[name]
}

func GetStringsLabel(name string) []string {
	value := GetStringLabel(name)
	if value == "" {
		return nil
	}
	return strings.Split(value, ",")
}

func GetBoolLabel(name string) bool {
	return lables[name] == "true"
}

```

Reply:
get it. Thanks a lot.